_posts/2020-02-19-cockpit-213.md

```diff
@@ -0,0 +1,54 @@
+---
+title: Cockpit 213
+author: mvo
+date: 2020-02-19
+tags: cockpit linux
+slug: cockpit-213
+category: release
+summary: Cockpit with documentation and virtual machine improvements
+comments: 'true'
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/). We release regularly. Here are the release notes from version 213.
+
+### Inline documentation
+
+Cockpit has started to include documentation for specific features,
+such as network bonds:
+
+![Feature documentation](/images/network-inline-docs.png){:.screenshot}
+
+We plan to expand this in the future and add many more short
+documentation popups.
+
+### Support for transient virtual machines
```

Comment:
Traditionally we have prefixed the sections with the affected page,  like "Machines:". I don't have a strong opinion about it, I consider it editorial freedom. Just mentioning it FTR.

Reply:
I think we don't need it, tbh, since the titles all mention "virtual machines", and the doc one is not about a specific page.